release/pkg/manager/operator/manager.go

```diff
@@ -272,6 +276,6 @@ func (o *OperatorManager) Clone() error {
 	return utils.Clone(fmt.Sprintf("git@github.com:%s/%s.git", o.githubOrg, o.repoName), o.branch, o.dir)
 }
 
-func (o *OperatorManager) make(target string, env []string) (string, error) {
-	return o.runner.Run("make", []string{"-C", o.dir, target}, env)
+func (o *OperatorManager) makeWithNoOutput(target string, env ...string) error {
```

Comment:
Why are we dropping the output from all of these commands? It just means we won't have debug logs available if something goes wrong no?